forge/forge/components/file_manager/file_manager.py

```diff
@@ -13,7 +16,21 @@
 logger = logging.getLogger(__name__)
 
 
-class FileManagerComponent(DirectiveProvider, CommandProvider):
+class FileManagerConfiguration(BaseModel):
+    files_path: str
```

Comment:
I don't like this name but also don't know what else it should be called

Reply:
`state_storage_directory`

Reply:
How about `resources_path`, `data_path`, storage_path`?
`state_storage_directory` isn't ideal because there may not only be agent state.

Reply:
`storage_base_path`?

Reply:
> `storage_base_path`

Does that imply `workspace_path` is a subdirectory of `storage_base_path`. It doesn't need to be for example multiple agents could have one workspace 😓

Reply:
I renamed it to `storage_path` for now